test/features/quicksearch_feature_test.rb

```diff
@@ -30,4 +30,22 @@ class QuickSearchFeatureTest < Capybara::Rails::TestCase
     all('.password-show')[0].click
     page.must_have_selector('.password-show', visible: false)
   end
+
+  test 'search and access account' do
+    login_as_user(:bob)
+    page.must_have_selector('.form-group.search')
+
+    fill_in 'q', with: 'account1'
+    page.must_have_selector('.account-entry')
+    all('.account-entry')[0].click
+    page.must_have_content('account1')
+  end
+
+  test 'search by get params' do
+    login_as_user(:bob)
+    visit('/search?q=account1')
+
+    page.must_have_selector('.account-entry')
```

Comment:
please check if the right account is shown. now any account could be returned and the test wouldn't fail.